### ISSUE TITLE
Get Final Predicted Labels:

### ISSUE THREAD

**Issue by lakshaymehra** [crux82/ganbert#15]
Hi @crux82 ,

I have run GANBERT on a variety of datasets and the results are very very promising. So, firstly, Thank you for this amazing paper and repository.

For further validation of the results, I would also like to fetch the final predicted labels as a .txt or .csv file from the model. 

I have tried modifying the ganbert.py to save the "predictions" from line 531 but this is a Tensor and its evaluation is not as simple as predictions.eval(). I have tried a lot of different methods but have not been able to extract the predicted labels.

Can you please guide me how I can extract the predicted labels from the model?

Thanks and Regards,

Lakshay.

**Comment by lakshaymehra** [comment]
Solved the issue, 

For those interested, you can create a custom formatter (eval_hooks_formatter function) and add an evaluation logging hook to output spec :

```
eval_logging_hook = tf.train.LoggingTensorHook({"d_loss": d_loss, "g_loss": g_loss, 
                                                 "per_example_loss": per_example_loss, 
                                                 "LOGITS" : logits, "LABEL": label_ids}, every_n_iter=1,formatter = eval_hooks_formatter)
```
